### PR TITLE
Add chrome.serial definitions

### PR DESCRIPTION
**Pull request by koloboid**
case 1. Add a new type definition.
- [ ] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [ ] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

### CODE REVIEW

**Comment by koloboid** [comment]
ups, implicitAny. will fix in few hours

**Comment by koloboid** [comment]
done, check it out, plz ;)

File: chrome/chrome-app.d.ts [hunk 1]
```diff
@@ -706,3 +706,115 @@ declare module chrome.usb {
...
+        connectionId: number;
+        data: ArrayBuffer;
+    }
+
+    interface ReceiveError {
+        connectionId: number;
+        error: ReceiveErrorType;
+    }
+
+    interface OnReceiveEvent extends chrome.events.Event<(info: ReceiveInfo)=> void> {}
```
Comment (RReverser) [inline chrome/chrome-app.d.ts]: `=> ` - please put space so that it would be both before and after.

File: chrome/chrome-app.d.ts [hunk 2]
```diff
@@ -706,3 +706,115 @@ declare module chrome.usb {
...
+    interface OnReceiveErrorEvent extends chrome.events.Event<(info: ReceiveError)=> void> {}
+
+    export var onReceive: OnReceiveEvent;
+    export var onReceiveError: OnReceiveErrorEvent;
+
+    export function getDevices(callback: (ports: Port[])=>void): void;
```
Comment (RReverser) [inline chrome/chrome-app.d.ts]: `=>` - no space here and below at all. Please fix for style consistency.

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**